src/app/components/EmbedConsentBanner/ConsentBanner.tsx

```diff
@@ -16,9 +16,11 @@ import { ConsentBannerProviders, getEventTrackingData } from '.';
 
 type BannerUrls = {
   cookiesUrl: {
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
Wonder is there a custom rule we could to the ESLint config to allow these. This is a pretty common pattern in TypeScript, so it would be a bit of a chore having to add in this disable-line each time.

Reply:
Yeah I struggled to find the right config / make sense of the docs: https://eslint.org/docs/latest/rules/no-unused-vars#options. 

Also - these are warnings only (and appear to have been introduced in https://github.com/bbc/simorgh/pull/12304/files - scroll down to the bottom), so they could technically be ignored as they don't cause the PR checks to fail... So we could do something about the error level perhaps?

Reply:
And it gives an error message like this:

```
'key' is defined but never used. Allowed unused vars must match /^_/u 
```

Reply:
See https://github.com/bbc/simorgh/pull/12206/commits/cbe49f57e2c0422915e404a3d4950c82fdbe701b

Reply:
Nice, follows the underscore for unused variable pattern 👍